end-to-end-test/local/docker_compose/keycloak.yml

```diff
@@ -2,11 +2,11 @@ version: '3'
 
 services:
   cbioportal:
+    image: cbioportal/cbioportal:demo-rfc72
```

Comment:
need to remove later, this is a hack b/c it's no longer picking up the `env/custom.sh` `BACKEND` parameter